### ISSUE TITLE
Default locale should not have a prefix

### ISSUE THREAD

**Issue by dottodot** [dealmore/terraform-aws-next-js#139]
I currently have the following in my next.config.js 
```
module.exports = {
  i18n: {
    locales: ['en'],
    defaultLocale: 'en'
  }
};
```
and locally the app runs without adding the /en prefix, however once deployed the site can only be accessed using `https://xxxxxxxxxx.cloudfront.net/en`

Is there any way to prevent this from happening so the site can be accessed without the prefix i.e `https://xxxxxxxxxx.cloudfront.net`

**Comment by ofhouse** [comment]
Hi,

generated routes seem to be correct but I noticed a bug in the proxy component that it doesn't handle the internal rewrite `/` -> `/en` correctly.

Will take a look, thanks for reporting!

**Comment by ofhouse** [comment]
This is now fixed in the [v0.9.1 release](https://github.com/dealmore/terraform-aws-next-js/releases/tag/v0.9.1).
